tests/actions.go

```diff
@@ -469,7 +479,42 @@ func (oa *operatorActions) UpgradeOperator(info *OperatorConfig) error {
 	if err != nil {
 		return fmt.Errorf("failed to upgrade operator to: %s, %v, %s", info.Image, err, string(res))
 	}
-	return nil
+
+	time.Sleep(5 * time.Minute)
```

Comment:
check ready status? that will be more efficient

Reply:
The main purpose here is to prevent the pod from being rebuilt after updating the operator. After updating the operator, it may become ready soon, but it may take a while to rebuild all the pods. This time is not easy to grasp, so I just use `sleep` here.

Reply:
I will change it to `wait.Poll`